test/yki/handler/base_test.clj

```diff
@@ -193,6 +193,9 @@
 (defn select-one [query]
   (first (select query)))
 
+(defn execute! [statement]
+  (jdbc/execute! @embedded-db/conn statement))
```

Comment:
En nyt tiedä maksaako vaivaa mutta search and replacella vois korvata nykyiset `jdbc/execute! @embedded-db/conn` tällä `execute!`:lla.